source/Stars.pde

```diff
@@ -0,0 +1,64 @@
+public class Stars extends PhysicsObj {
+  
+  private PVector target = new PVector();
+  private PVector startPos;
+  private float kX;
+  private float kY;
+  private float Size;
+  private float visibleSize = 2;
+  
+  public Stars(PVector dest, PVector start) {
+    super(start);
+    startPos = start.copy();
```

Comment:
Considering that you're initiating a new Vector for each Star spawned, copying the object isn't desirable.